source/docs/software/labView/resources/local-and-global-variables-in-labview-for-frc.rst

```diff
@@ -0,0 +1,47 @@
+Local and Global Variables in LabVIEW for FRC
+=============================================
+
+.. image::images/ni-logo.png
+
+This example serves as an introduction to local and global variables, how they are used in the default LabVIEW for FRC Robot Project, and how you might want to use them in your project.
+
+Local variables and global variables may be used to transfer data between locations within the same VI (local variables) or within different VI’s (global variables), breaking the conventional `Data Flow Paradigm <http://www.ni.com/getting-started/labview-basics/dataflow>`__ for which LabVIEW is famous.  Thus, they may be useful when, for whatever reason, you cannot wire the value directly to the node to another.
+
+Note: One possible reason may be that you need to pass data between consecutive loop iterations; Miro\_T covered this `in this post <http://decibel.ni.com/content/docs/DOC-41137>`__.  It should also be noted that the `feedback node <http://zone.ni.com/reference/en-XX/help/371361L-01/lvconcepts/block_diagram_feedback/>`__ in LabVIEW may be used as an equivalent to the shift register, although that may be a topic for another day!
+
+Introduction to Local and Global Variables
+------------------------------------------
+
+Local variables may be used within the same VI.  Create a local variable by right-clicking a control or indicator on your Front Panel:
+
+.. image::images/local-and-global-variables-in-labview-for-frc/1.png
+
+You may create a local variable from the Structures palette on the block diagram as well.  When you have multiple local variables in one VI, you can left-click to choose which variable it is:
+
+.. image::images/local-and-global-variables-in-labview-for-frc/2.png
+
+Global variables are created slightly differently.  Add one to the block diagram from the Structures palette, and notice that when you double-click it, it opens a separate front panel.  This front panel does not have a block diagram, but you add as many entities to the front panel as you wish and save it as a \*.vi file:
+
+.. image::images/local-and-global-variables-in-labview-for-frc/3.png
+
+.. note:: Note: Be very careful to avoid race conditions when using local and global variables!  Essentially, make sure that you are not accidentally writing to the same variable in multiple locations without a way to know to which location it was last written.  For a more thorough explanation, see `this help document <http://zone.ni.com/reference/en-XX/help/371361L-01/lvconcepts/using_local_and_global/>`_
```

Comment:
Remove the redundant "Note" here. Not the directive, just the word at the beginning of the paragraph.